plasmapy/diagnostics/charged_particle_radiography.py

```diff
@@ -19,6 +20,7 @@
 
 from plasmapy import particles
 from plasmapy.formulary.mathematics import rot_a_to_b
+from plasmapy.formulary.relativity import relativistic_energy
```

Comment:
```suggestion
```

Since `relativistic_energy` is no longer used in this module.

Reply:
:cry: Don't worry #979, we won't forget you!